server/src/main/resources/db/V4__create_pulse_response_table.sql

```diff
@@ -0,0 +1,9 @@
+drop table if exists pulse_response;
+CREATE TABLE pulse_response (
+   id varchar PRIMARY KEY,
+   submissionDate date,
+   updatedDate date,
+   teamMemberId varchar REFERENCES member_profile(uuid),
+   internalFeelings varchar,
+   externalFeelings varchar
+);
```

Comment:
I hate to be the bearer of bad news...but you're gonna have to change this to V5

Reply:
Do this before you merge...I'll approve anyway.

Reply:
I've renamed the file to V5